packages/yourloops/pages/training/training.tsx

```diff
@@ -124,22 +134,55 @@ function TrainingPage(): JSX.Element {
               </Box>
             </CardMedia>
             <CardContent className={classes.cardContent}>
-              <Trans
-                i18nKey="training-body"
-                t={t}
-                components={{ trainingLink }}
-                values={{ training }}>
-                New {{ training }} available
-              </Trans>
+              {trainingOpened
+                ? <FormControlLabel
+                  id={'form-control-training'}
+                  control={
+                    <Checkbox
+                      id={'checkbox-training'}
+                      aria-label={t('training-checkbox')}
+                      className={classes.checkbox}
+                      checked={checked}
+                      onChange={() => {
+                        setChecked(true)
+                      }}
```

Comment:
Could be on one line (better readability)
```suggestion
                      onChange={() => setChecked(true)}
```